org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/ui/AgentUi.java

```diff
@@ -140,4 +138,22 @@ private boolean loadAgent(String agentJar, String xmlPath, String pid) {
 		return true;
 	}
 
+	private MBeanServerConnection initMBeanServerConnection() {
```

Comment:
For the name`initMBeanServerConnection`, *init* implies a change in state, often creating a connection and saving it somewhere (eg. member variables). Since one `AgentUi` handles an agent on only one JVM, is think it makes sense to have a `MBeanServerConnection` member variable (ie. `private void initMBeanServerConnection()`).

Alternative, make a util method `private MBeanServerConnection initMBeanServerConnection(VirtualMachine vm)` and leave it to the caller the decided what to do with the newly established connection?

Reply:
Fixed in https://github.com/rh-jmc-team/jmc-agent-plugin/pull/16/commits/345bcf082d098ca54f053d0c6c432b098aca78ea